src/demos/App.js

```diff
@@ -35,6 +36,11 @@ const demos = {
     text: 'The CERN official platform for videos.',
     cmp: <CERNVideosReactSearchKit />,
   },
+  'namespaced_example': {
+    label: "video.cern.ch namespacing example",
```

Comment:
Just to keep it shorter:
```suggestion
    label: "video.cern.ch namespaced",
```